modules/RTC/JitsiLocalTrack.js

```diff
@@ -28,26 +29,92 @@ function JitsiLocalTrack(stream, videoType,
             this.dontFireRemoveEvent = false;
         }.bind(this));
     this.initialMSID = this.getMSID();
+    this.inMuteOrUnmuteProcess = false;
 }
 
 JitsiLocalTrack.prototype = Object.create(JitsiTrack.prototype);
 JitsiLocalTrack.prototype.constructor = JitsiLocalTrack;
 
 /**
+ * Mutes the track. Will reject the Promise if there is mute/unmute operation
+ * in progress.
+ * @returns {Promise}
+ */
+JitsiLocalTrack.prototype.mute = function () {
```

Comment:
I see what you did this, but since ``mute`` and ``unmute`` look identical (except for a parameter), I was wondering if it is worthy to extract another function for the common logic?